defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # Set the version of Node.js to install (8.x", "10.x", "12.x", "13.x", etc.).
```

Comment:
I was actually going to open a similar PR and saw you already did this. Thanks @jtcmedia 🙏  

Small nit-picky thing, just thought it would make sense to fix, since it is missing a `"`, and update the comment too. 
```suggestion
# Set the version of Node.js to install ("12.x", "13.x", "14.x", "15.x", etc.).
```

Reply:
Thanks @KFoxder :smiley: